app/workers/export_photos.rb

```diff
@@ -5,7 +5,7 @@
 
 module Workers
   class ExportPhotos < Base
-    sidekiq_options queue: :export
+    sidekiq_options queue: :low
```

Comment:
I'd give this a `medium`, since while long running, it is an action on which completion the user actively waits for.